mobile/src/components/course/components/course-short-description/course-short-description.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+
+import { CourseDescriptionFirstWordsCount } from '~/common/enums/enums';
+import { Content, Pressable, Text, View } from '~/components/common/common';
+import {
+  getTextWithoutHTMLTags,
+  getWordsCountFromString,
+} from '~/helpers/helpers';
+
+import { styles } from './styles';
+
+type Props = {
+  description: string;
+  handleSeeMore: () => void;
+  width: number;
+};
+
+const CourseShortDescription: FC<Props> = ({
```

Comment:
We also should have see less button, so I think we can rename component to CourseDescription and pass handler to toggle expand state

Reply:
Why do we need 'See less'? Because, if user has already scrolled to the end of description, why he need to hide it? Or maybe we should make 'See less' btn fixed on the screen and don't show it in the end? 
Also when we hide description the modules are shown somewhere in the middle of the list (the video is attended)

https://user-images.githubusercontent.com/89215185/188275957-ad52f055-2391-4c65-bbf0-f3fd31fe6ed7.mp4

I thought about scrolling automatically to the top, after press 'see less', but only ways I found are useing useRef and in our case ScrollView, as I understand, is avoiding useRef.

Reply:
It's good when user have an ability to return to the previous state, let's do this without scroll to the top for now

Reply:
The behavior on the record is ok

Reply:
ok